dvc/repo/__init__.py

```diff
@@ -360,14 +391,38 @@ def graph(self, stages=None, from_directory=None):
 
         return G, G_active
 
-    def pipelines(self, from_directory=None):
+    @cached_property
+    def _graph(self):
+        return self._collect_graph()
+
+    @property
+    def graph(self):
+        return self._graph[0]
```

Comment:
could we use names tuple or something to make it explicit that 1 is active, 0 is not. Otherwise it's easy to confuse with the first DAG, first pipeline, etc